NAS-100746: Review/update TrueNAS Support page (#1423)
freenas/freenas-docs: userguide/system.rst

supported hardware.

If the license expires or additional hardware, features, or
contract type are required,
:ref:`contact iXsystems Support <Contacting iXsystems>`. After a new
license has been provided, click :guilabel:`Update License`, paste in
the new license, and click :guilabel:`SAVE LICENSE`. The page updates to
show the new license details.

:guilabel:`User Guide (PDF)` opens a new browser tab to the iXsystems
%brand%
`Information Library <https://www.ixsystems.com/blog/knowledgebase_category/truenas/>`__.
The %brand% User Guide, product datasheets, %brand% hardware setup
guides, and task assistance articles are all available in this library.

To generate a support ticket, fill in the fields:

* :guilabel:`Name` is the name of the person the iXsystems Support
  Representative should contact to assist with the issue.

* :guilabel:`Email` is the email address of the person to contact.

* :guilabel:`Phone` is the phone number of the person to contact.

> * :guilabel:`Type` is a drop-down menu to select whether the ticket is

Reviewer: ```suggestion
* :guilabel:`Type` is a drop-down menu to select the ticket type:
```

>   to report a software bug, report a hardware failure, ask for

Reviewer: ```suggestion
  a software bug, a hardware failure, a request for
```

>   assistance in installing or configuring the system, or request

Reviewer: "in installing" sounds little odd.
```suggestion
  help with installing or configuring the system, or a request
```

>   assistance in diagnosing a performance bottleneck.

Reviewer: ```suggestion
  help with diagnosing a performance bottleneck.
```
